app/components/new-signup/checkbox.js

```diff
@@ -0,0 +1,37 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { LABEL_TEXT } from '../../constants/new-signup';
+
+export default class SignupComponent extends Component {
+  checkboxData = [
```

Comment:
Can we move it to constant?

Reply:
+1 pleas move this to constansts

Reply:
Done with changes...can you please check my latest commit